utensor_cgen/__main__.py

```diff
@@ -70,6 +70,9 @@ def _build_parser():
   parser.add_argument("--debug-comment", dest="debug_cmt",
                       action="store_true",
                       help="Add debug comments in the output source file (default: %(default)s)")
+  parser.add_argument("-Q", "--quantize", dest="quantize_flag",
+                      action="store_true", default=False
```

Comment:
You missed a `,` after `default=False`.

Reply:
And `default=False` is not necessary if `action="store_true"` is given.

Reply:
fixed